report_thesis/src/sections/background.tex

```diff
@@ -148,6 +150,103 @@ \subsubsection{Partial Least Squares (PLS)}
 The components are then used to predict the target variable by fitting a linear model via least squares regression.
 
 \subsubsection{Support Vector Regression (SVR)}
+\gls{svr} is a regression technique that extends the principles of \gls{svm} to regression problems.
+We therefore provide an overview of \gls{svm}s based on \citet{James2023AnIS} before discussing \gls{svr}s.
+
+\gls{svm} is a supervised learning algorithm used primarily for classification tasks.
+A core concept in \gls{svm} is the \textit{hyperplane}.
+Generally, a hyperplane is a subspace of one dimension less than its ambient space.
+This means that in a two-dimensional space, a hyperplane is a line, while in a three-dimensional space, it is a plane, and so on.
+
+\gls{svm} is built on the idea of finding the hyperplane that best separates the data points into different classes.
+This hyperplane is chosen to maximize the margin, which is the distance between the hyperplane and the nearest data point from either class.
+The data points that lie closest to the hyperplane are called \textit{support vectors}.
```

Comment:
Support vectors are not just any points close to the hyperplane; they are specifically the points that lie on or within the margin.

```suggestion
The instances right on or inside the margin are called \textit{support vectors}, which are used to 'support' the margin and decision boundary.
```

Reply:
What do you mean by 'instances' in the suggestion?

Reply:
'Instance' is a general ML term for each training example, e.g., the rows in our masked avg. intensity matrix.